renovate.json

```diff
@@ -36,14 +36,9 @@
       "matchDatasources": ["orb"],
       "matchPackageNames": ["cimg/redis"],
       "allowedVersions": "6.2"
-    },
-    {
-      "matchPackageNames": ["typescript"],
-      "rangeStrategy": "bump"
-    },
-    {
-      "matchPackageNames": ["govuk-frontend"],
-      "rangeStrategy": "bump"
     }
-  ]
+  ],
+  "vulnerabilityAlerts": {
+    "stabilityDays": 0
```

Comment:
As per [Renovate docs](https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts)

> Renovate can read GitHub's Vulnerability Alerts to customize its Pull Requests

So configure Renovate to raise PRs for these straight away